app/views/shared/_header.html.erb

```diff
@@ -30,7 +30,7 @@
             <ul class="dropdown-menu">
               <% Environment.all.each do |env| %>
                 <li>
-                  <%= link_to dashboard_path(env) do %>
+                  <%= link_to dashboard_path(env.name) do %>
```

Comment:
Aye, switching to use to_param override will necessitate me changing all the admin pages as well for environments. But I can do that.